ios/Video/RCTVideo.swift

```diff
@@ -306,7 +306,7 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
             _plugin?.fireOfflineEvents()
         }
 
-        _plugin?.adapter = YBAVPlayerAdapter.init(player: _player!) as? YBPlayerAdapter<AnyObject>
+        _plugin?.adapter = YBAVPlayerAdapterSwiftTranformer.transform(from: YBAVPlayerAdapter(player: _player!))
```

Comment:
not sure how this stopped working all of a sudden 🥲 it used to work unless we changes something?